# FloatArray2D.msg
float32[3] point0
float32[8] uwb1
float32[8] uwb2
float32[8] uwb3
float32[8] uwb4
float32[8] uwb5
float32[8] uwb6
float32[8] uwb7


